src/build.js

```diff
@@ -2,12 +2,12 @@ import {execCommand} from './utils';
 import {setApplitoolsId} from './applitoolsScripts';
 import {install} from './install';
 
-export function build(buildType) {
+export async function build(buildType) {
   if (process.env.APPLITOOLS_GITHUB_FT) {
     setApplitoolsId();
   }
 
-  install();
+  await install();
   execCommand('npm run build --if-present');
```

Comment:
do we want to `await` on this?

Reply:
That's the old sync command, so it's already sync